Refusing to install package with name "amazon-chime-sdk-js"

On a fresh Amazon Linux EC2 instance:

```
[ec2-user@ip-10-0-101-221 ~]$ git clone https://github.com/aws/amazon-chime-sdk-js.git
Cloning into 'amazon-chime-sdk-js'...
remote: Enumerating objects: 262, done.
remote: Counting objects: 100% (262/262), done.
remote: Compressing objects: 100% (190/190), done.
remote: Total 1685 (delta 69), reused 200 (delta 51), pack-reused 1423
Receiving objects: 100% (1685/1685), 838.40 KiB | 1.88 MiB/s, done.
Resolving deltas: 100% (708/708), done.

[ec2-user@ip-10-0-101-221 ~]$ cd amazon-chime-sdk-js/

[ec2-user@ip-10-0-101-221 amazon-chime-sdk-js]$ node -v
v13.3.0

[ec2-user@ip-10-0-101-221 amazon-chime-sdk-js]$ npm -v
6.13.1

[ec2-user@ip-10-0-101-221 amazon-chime-sdk-js]$ npm install amazon-chime-sdk-js --save
npm ERR! code ENOSELF
npm ERR! Refusing to install package with name "amazon-chime-sdk-js" under a package
npm ERR! also called "amazon-chime-sdk-js". Did you name your project the same
npm ERR! as the dependency you're installing?
npm ERR!
npm ERR! For more information, see:
npm ERR!     <https://docs.npmjs.com/cli/install#limitations-of-npms-install-algorithm>

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ec2-user/.npm/_logs/2019-12-05T05_44_16_302Z-debug.log
```

--- a/README.md
+++ b/README.md
@@ -31,6 +31,9 @@ To build, test, and run demos from source you will need:
 
 ### Installing from NPM
 
+To add the Amazon Chime SDK for JavaScript into an existing application,
+install the package directly from npm:
+
 ```
 npm install amazon-chime-sdk-js --save
 ```

